.github/workflows/automerge.yml

```diff
@@ -0,0 +1,41 @@
+name: automerge
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - reopened
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      if: ${{ github.event.label.name == 'automerge' }}
+      uses: actions/checkout@v1
+    - name: Check status
+      if: ${{ github.event.label.name == 'automerge' }}
+      run: bash scripts/status-checker.sh ${GITHUB_REPOSITORY} ${GITHUB_RUN_ID}
+    - name: Generate token
+      if: ${{ github.event.label.name == 'automerge' }}
```

Comment:
It's because the JWT will expire ten min after creation, so then if the workflow has to re-run it would need to wait for status checks and e2e's take ~15-20 min